charts/stash-enterprise/templates/mutating-webhook.yaml

```diff
@@ -140,4 +140,25 @@ webhooks:
 {{- if and (ge $major 1) (ge $minor 12) }}
   sideEffects: None
 {{- end }}
+- name: restoresession.admission.stash.appscode.com
```

Comment:
What are we mutating?

Reply:
We are migrating `RestoreSession`.

https://github.com/stashed/apimachinery/blob/b8c42c18d61fed4cb4ed7dde6b9ba496066636b2/apis/stash/v1beta1/restore_session_helpers.go#L51